### DISCUSSION TITLE
关于丢包的推送

### DISCUSSION THREAD

**Discussion by StephenSSR** [zdz/ServerStatus-Rust#25] [Q&A] [answered]
请问大佬能否添加一个关于丢包超过20%的推送通知呢？

**Comment by zdz** [comment, marked as answer]
你可以在 custom_tpl 模板后面追加以下内容实现

```
{% if host.ping_10086 > 20  %}
<pre>❗{{ host.name }} 主机中移动丢包率超20%, 当前{{ host.ping_10086 }}% </pre>
{% endif %}
```

电信是 ping_189
联通是 ping_10010

**Reply by StephenSSR** [reply]
好的，多谢大佬解答

**Comment by StephenSSR** [comment]
不好意思大佬，再请问如何启用custom_tpl呢？

**Comment by zdz** [comment]
custom_tpl 在配置文件 config.toml 里面，改完重启一下 server 就生效了

**Comment by StephenSSR** [comment]
是的大佬我看到了，我是想问custom_tpl是默认启用的吗？例如模板里的内存使用率和硬盘使用率，因为我发现我在添上您关于丢包的那部分内容后重启，并没有推送关于丢包超过20%的通知

**Comment by zdz** [comment]
配置上就会启用，通知现在是每隔 x 秒（配置文件里的 notify_interval ）检查一次，要在那一秒内检测到才会发通知
